hackathon.html

```diff
@@ -0,0 +1,102 @@
+<script type="text/javascript" src="js/handlebars-v4.0.10.js"></script>
+<div class="container">
+<div id="nixcon-logo">
+  <img src="/images/nixos-logo.png" class="responsive-img nixos-logo" width="300" style="margin-top: 2em; margin-bottom: 0em;" />
+</div>
+
+  <div class="row">
+    <div class="col s12 m12">
+      <h1 class="header center grey-text text-darken-2">NixCon Hackathon</h1>
+      <h3 class="header center grey-text text-darken-2">Part of the <a href="./">NixCon 2017</a></h3>
+    </div>
+  </div>
+  <br/>
+</div>
+
+<div class="container">
+  <div class="row">
+    <div class="col s12 m12">
+      <h3 class="header center grey-text text-darken-2">Topics</h3>
+    </div>
+  </div>
+  <div class="row" id="topics">
+  </div>
+  <div class="row">
+    <div class="col s12 m12">
+        <h5>Want to propose your own topic?</h5>
+        You can propose new topics via the following workflow:
+        <ol>
+            <li>fork the page repository <a href="https://github.com/cko/nixcon2017">cko/nixcon2017</a></li>
+            <li>add your topic to the file <tt>./hackathon-topics.json</tt>
+                <ul>
+                    <li>the skill level range goes from 1 (Beginner) to 3 (Expert)</li>
+                </ul>
+            </li>
+            <li>create a pull request</li>
+        </ol>
+    </div>
+  </div>
+</div>
+<script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+<script>
+ $( document ).ready(function() {
+    function skillLevelToHTML(lvl) {
+        switch(lvl) {
+            case 1: return "Beginner";
+            case 2: return "Advanced";
+            case 3: return "Expert";
+            default: return "?";
+        }
+    }
+
+    function skillLevelsToHTML(lvl1, lvl2) {
+        if (lvl1 === lvl2) {
+            return skillLevelToHTML(lvl1);
+        }
+        return skillLevelToHTML(lvl1) + " - " + skillLevelToHTML(lvl2)
+    }
+
+    $.getJSON( "./hackathon-topics.json", function( data ) {
+        function compare(a,b) {
+            if (a.title < b.title)
+                return -1;
+            if (a.title > b.title)
+                return 1;
+            return 0;
+        }
+        data.sort(compare);
+
+        var items = [];
+        $.each( data, function(key, val) { items.push(
+            val.host +
+            "<h4>" + val.title + "</h4>" +
+            "<b> Skill Level: " + skillLevelsToHTML(val.skillLevel, val.skillLevelTo) + "</b></br>" +
+            '<a href="' + val.homepage + '">' + val.homepage + '</a></br>' +
+            val.description
+        );});
+        $("#topics").append('<div class="col s12 m6"><div class="card">' + items.join( '</div></div><div class="col s12 m6"><div class="card">' ) + '</div></div>');
+    });
```

Comment:
Would be cool, if you would use a template instead of making the already pretty hacky page even more hackier